src/main/js/field/LabeledField.js

```diff
@@ -21,7 +21,14 @@ class LabeledField extends Component {
 
         const baseClasses = `${labelClass} text-right`;
         const labelClasses = (required) ? `${baseClasses} required` : baseClasses;
-
+        let severity = 'ERROR';
+        let fieldMessage = '';
```

Comment:
These can be converted into const if you using a ternary operator like you did on line 30 and 31